### PR TITLE
Update: TUitTUWien.TUtoolbox version 2021.0.0.338

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/53651)

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]

**Review by OfficialEsco — DISMISSED** [review]

File: manifests/t/TUitTUWien/TUtoolbox/2021.0.0.338/TUitTUWien.TUtoolbox.yaml [hunk 1]
```diff
@@ -1,22 +1,8 @@
...
-    InstallerUrl: "https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox/versions/TUToolBox_2021_0_0_338.exe"
-    InstallerSha256: "636023555d427cffc1a6086af4f06d0d05748ae6ef4bb8fe7925836eab26bca7"      
-ManifestType: "singleton"
-ManifestVersion: "1.0.0"
\ No newline at end of file
+# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: TUitTUWien.TUtoolbox
+PackageVersion: 2021.0.0.338
```
Comment (ItzLevvie) [inline manifests/t/TUitTUWien/TUtoolbox/2021.0.0.338/TUitTUWien.TUtoolbox.yaml:5]:
```suggestion
PackageVersion: 2021.0.0.399
```

File: manifests/t/TUitTUWien/TUtoolbox/2021.0.0.338/TUitTUWien.TUtoolbox.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+UpgradeBehavior: install
+InstallerSwitches:
+  Custom: /SUPPRESSMSGBOXES
+Installers:
+- Architecture: x64
+  InstallerUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox/versions/TUToolBox_2021_0_0_338.exe
+  InstallerSha256: 636023555D427CFFC1A6086AF4F06D0D05748AE6EF4BB8FE7925836EAB26BCA7
+- Architecture: x86
+  InstallerUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox/versions/TUToolBox_2021_0_0_369.exe
+  InstallerSha256: 8BB584E0E5D220EF1FCCC8549C46472F5EA1886A558C310079AB335D683BD6AE
```
Comment (ItzLevvie) [inline manifests/t/TUitTUWien/TUtoolbox/2021.0.0.338/TUitTUWien.TUtoolbox.installer.yaml:20]:
```suggestion
  InstallerUrl: https://service.it.tuwien.ac.at/swsupport/tools/tutoolbox/versions/TUToolBox_2021_0_0_338.exe
  InstallerSha256: 636023555D427CFFC1A6086AF4F06D0D05748AE6EF4BB8FE7925836EAB26BCA7
```

File: manifests/t/TUitTUWien/TUtoolbox/2021.0.0.338/TUitTUWien.TUtoolbox.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: TUitTUWien.TUtoolbox
+PackageVersion: 2021.0.0.338
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
```
Comment (ItzLevvie) [inline manifests/t/TUitTUWien/TUtoolbox/2021.0.0.338/TUitTUWien.TUtoolbox.installer.yaml:11]:
```suggestion
UpgradeBehavior: install
InstallerSuccessCodes:
- 1
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by vedantmgoyal2009** [comment]
@ItzLevvie I think there is a typo in `PackageVersoin` suggestion.
image

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**